### PR TITLE
Added Manual Toggle button to switch dark mode

### PR DESCRIPTION
**Pull request by Interstellar07**
This is the PR of the issue https://github.com/AyushAgnihotri2025/Calculator/issues/10

Changes proposed
✅Added Manua(Toggle Buttom)l + System Wide  Dark Mode Support.

Check List (Check all the applicable boxes)
✅ My code follows the code style of this project.
✅ All new and existing tests passed.
✅ This PR does not contain plagiarized content.
✅ The title of my pull request is a short description of the requested changes.
Sc
SS2
reenshots

SS$

SS5
SS1
SS6

### CODE REVIEW

**Comment by Interstellar07** [comment]
@AyushAgnihotri2025 Kindly look into this PR.

**Review by AyushAgnihotri2025 — APPROVED** [review]
Nice work 👍 @Interstellar07 , Keep It Up.

**Comment by AyushAgnihotri2025** [comment]
Hey @Interstellar07 👋, 

Kindly recheck your commit once again as it looks like you might have missed out on some files while committing them.

So, kindly see you ASAP and create a new pull with updated files.

**Comment by AyushAgnihotri2025** [comment]
Till then, I have reverted your pull.

File: app/src/main/res/menu/side_menu.xml [hunk 1]
```diff
@@ -9,9 +9,13 @@
             android:icon="@drawable/ic_share"
             android:title="Share"
             app:showAsAction="always"/>
+
         <item android:id="@+id/feedback"
             android:icon="@drawable/ic_feedback"
             android:title="Feedback" />
+        <item android:id="@+id/preferences"
+            android:icon="@drawable/ic_baseline_settings_24"
```
Comment (AyushAgnihotri2025) [inline app/src/main/res/menu/side_menu.xml:17]: This drawable asset is missing. So, upload as per your required size.

Comment (Interstellar07) [reply app/src/main/res/menu/side_menu.xml:17]: Extremely sorry for this, I am on it to resolve it at earliest.

Comment (Interstellar07) [reply app/src/main/res/menu/side_menu.xml:17]: Hi @AyushAgnihotri2025 ,
I have raised a PR of missing files, Kindly verify and merged this PR and the earlier raised PR together.

Comment (AyushAgnihotri2025) [reply app/src/main/res/menu/side_menu.xml:17]: No problem buddy.

**Pull request merged**